docs/sql-ref-datetime-pattern.md

```diff
@@ -199,11 +231,15 @@ The count of pattern letters determines the format.
 
 - Number/Text: If the count of pattern letters is 3 or greater, use the Text rules above. Otherwise use the Number rules above.
 
-- Fraction: Outputs the micro-of-second field as a fraction-of-second. The micro-of-second value has six digits, thus the count of pattern letters is from 1 to 6. If it is less than 6, then the micro-of-second value is truncated, with only the most significant digits being output.
+- Fraction: Use `'S..S'` pattern for parsing and formatting fraction of second.
+  For parsing, the acceptable fraction length can be [1, `'S..S'`.length(<=9)] and with `[]` surrounded can be [0, `'S..S'`.length(<=9)].
```

Comment:
`[1, number of contiguous 'S']`

Reply:
And we don't need to mention `[]`, as it's semantic is well defined and orthogonal to the second fraction.